scripts/ci/kubernetes/docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+#  Licensed to the Apache Software Foundation (ASF) under one   *
+#  or more contributor license agreements.  See the NOTICE file *
+#  distributed with this work for additional information        *
+#  regarding copyright ownership.  The ASF licenses this file   *
+#  to you under the Apache License, Version 2.0 (the            *
+#  "License"); you may not use this file except in compliance   *
+#  with the License.  You may obtain a copy of the License at   *
+#                                                               *
+#    http://www.apache.org/licenses/LICENSE-2.0                 *
+#                                                               *
+#  Unless required by applicable law or agreed to in writing,   *
+#  software distributed under the License is distributed on an  *
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+#  KIND, either express or implied.  See the License for the    *
+#  specific language governing permissions and limitations      *
+#  under the License.                                           *
+
+FROM ubuntu:16.04
```

Comment:
Just checking: would this work with alpine as well? As Ubuntu is quite large

Reply:
For what it's worth, we use alpine as our base and it's been working pretty well. We're planning on swapping in the KubernetesExecutor in, so some things will be changing, but here's our current Dockerfile.
https://github.com/astronomerio/astronomer/blob/master/docker/platform/airflow/Dockerfile